server/bleep/src/webserver/answer.rs

```diff
@@ -452,13 +453,15 @@ impl<'s> AnswerAPIClient<'s> {
         prompt: &str,
         max_tokens: u32,
         temperature: f32,
+        provider: &str,
```

Comment:
Can we update this to use the `Provider` enum, as implemented in bloopai/answer-api#17?

See [46be9b1](https://github.com/BloopAI/answer-api/commit/46be9b1d823f60d6505cd37d9dde586c1031922d) for details.